example/example.html

```diff
@@ -15,7 +15,10 @@
         console.log(Module);
         window.addEventListener('artoolkitplus-loaded', function (e) {
             const tracker = loadCalib("data/PGR_M12x0.5_2.5mm.cal", function (tracker) {
+                // set the PixelFormat
                 tracker.setPixelFormat(Module.PIXEL_FORMAT.PIXEL_FORMAT_LUM.value);
+                //check if the PixelFormat is correct
+                console.log(tracker.getPixelFormat())
```

Comment:
This will print:
`PIXEL_FORMAT_PIXEL_FORMAT_RGB()`
but we set to `PIXEL_FORMAT_LUM` so what is going on here?